sql/2021/performance/LCP_element.sql

```diff
@@ -0,0 +1,13 @@
+#standardSQL
+# LCP element node
+
+SELECT
+  _TABLE_SUFFIX AS client,
+  JSON_EXTRACT(payload, "$._performance.lcp_elem_stats[0].nodeName") AS lcp_node,
+  COUNT(DISTINCT url) AS total_urls,
+FROM
+  --`httparchive.pages.2021_07_01_*`
+  `httparchive.sample_data.pages_*`
+GROUP BY
+ client,
+ lcp_node
```

Comment:
I'd also recommend sorting the results so the largest nodes are at the top
```suggestion
GROUP BY
  client,
  lcp_node
ORDER BY
  pct DESC
```